src/widgets/search-box/__tests__/search-box-test.js

```diff
@@ -411,8 +411,8 @@ describe('searchBox()', () => {
         simulateInputEvent('test', 'tes', widget, helper, state, container);
       });
 
-      it('updates the query', () => {
-        expect(helper.setQuery.calledOnce).toBe(true);
+      it('does not update the query in the state', () => {
+        expect(helper.setQuery.calledOnce).toBe(false);
```

Comment:
Should be based on callCount instead of callOnce to enforce that it's actually never called (and call count has the nice effect of comparing numbers)